### PR TITLE
Лахтачев -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by Yaroslav-Lahtachev**
Added more unit tests

### CODE REVIEW

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/LahtachevYaroslavTest.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+import static org.junit.Assert.*;
+
+public class LahtachevYaroslavTest {
+
+    private final double delta = 0.001;
+
+    @Test
+    public void defaultConstructorMakeZeroNumber() {
+        ComplexNumber complexNumber = new ComplexNumber();
```
Comment (Selllavi) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/LahtachevYaroslavTest.java:14]: maybe will be better rename test "canCreateComplexNumberWithoutInitialValues"?

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/LahtachevYaroslavTest.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        assertEquals(new ComplexNumber(0, 0), complexNumber);
+    }
+
+    @Test
+    public void constructorStringAndDoubleReturnEqualNumber() {
+        ComplexNumber complexNumber1 = new ComplexNumber("1", "1");
```
Comment (Selllavi) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/LahtachevYaroslavTest.java:20]: strange name

Comment (Yaroslav-Lahtachev) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/LahtachevYaroslavTest.java:20]: may be like "areNumbersWithDifferentInitialTypesEqual"?

Comment (Selllavi) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/LahtachevYaroslavTest.java:20]: yeah, it's nice:)

**Comment by Yaroslav-Lahtachev** [comment]
@Selllavi, @ViolettaShirokikh can you review my super-puper tests?

**Comment by Selllavi** [comment]
👍

**Comment by ViolettaShirokikh** [comment]
good :+1:

**Pull request merged**